### PR TITLE
CLOUDP-79093: [mongocli] move to github actions

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-79093

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,7 +83,7 @@ e2e-test: build ## Run E2E tests
 .PHONY: integration-test
 integration-test: ## Run integration tests
 	@echo "==> Running integration tests..."
-	${TEST_CMD} --tags="${INTEGRATION_TAGS}" ./internal...
+	${TEST_CMD} --tags="${INTEGRATION_TAGS}" -count=1 ./internal...
```
Comment (gssbzn) [inline Makefile:86]: avoid cache

File: internal/config/profile.go [hunk 2]
```diff
@@ -313,7 +313,7 @@ func (p *Profile) Delete() error {
 }
 
 func (p *Profile) Filename() string {
-	return fmt.Sprintf("%s/%s.toml", p.configDir, ToolName)
+	return filepath.Join(p.configDir, ToolName+".toml")
```
Comment (gssbzn) [inline internal/config/profile.go:316]: I'm not sure if this was a bug for windows or not but here we are

File: internal/config/profile_integration_test.go [hunk 3]
```diff
@@ -30,15 +31,18 @@ const (
 
 func testProfile(profileContents string) *Profile {
 	fs := afero.NewMemMapFs()
-	testConfigDir := "/test"
+	testConfigDir, _ := filepath.Abs("test")
```
Comment (gssbzn) [inline internal/config/profile_integration_test.go:34]: making this works for windows tests

**Review by threebee — APPROVED** [review]
LGTM

**Pull request merged**